### PR TITLE
Updated it to work on crystal 0.2

### PR DESCRIPTION
**Pull request by danieltanfh95**
Changes: 

1. Tuple `:` changed to Hash `=>`

2. added explicit `to_i` so that port is passed as integer.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,6 +1,6 @@
-/doc/
-/libs/
-/.crystal/
-/.shards/
-
-
+/doc/
+/libs/
```
Comment (sdogruyol) [inline .gitignore:2]: Please fix line endings

**Comment by sdogruyol** [comment]
Thank you 👍

File: .travis.yml [hunk 2]
```diff
@@ -1 +1 @@
-language: crystal
+language: crystal
```
Comment (sdogruyol) [inline .travis.yml]: Please fix line endings

File: shard.yml [hunk 3]
```diff
@@ -1,7 +1,7 @@
-name: fast-http-server
-version: 0.4.0
-
-authors:
-  - Serdar Dogruyol <dogruyolserdar@gmail.com>
-
-license: MIT
+name: fast-http-server
```
Comment (sdogruyol) [inline shard.yml]: Please fix line endings

File: shard.yml [hunk 4]
```diff
@@ -1,7 +1,7 @@
...
+version: 0.4.0
+
+authors:
+  - Serdar Dogruyol <dogruyolserdar@gmail.com>
```
Comment (sdogruyol) [inline shard.yml]: Please fix line endings

File: shard.yml (hunk 3 above)
Comment (danieltanfh95) [reply shard.yml]: Do you know of a good way to fix all line endings? I already tried to change them to unix style but some still have it

Comment (sdogruyol) [reply shard.yml]: IDK, it's probably your OS / Editor dependent

**Comment by danieltanfh95** [comment]
hmm, I've already edited them again in vim on ubuntu, so I'm not sure
what's going on here, maybe you can take a look, if you're on linux?

**Comment by sdogruyol** [comment]
@Shadowys i'm using OS X

**Comment by danieltanfh95** [comment]
@sdogruyol Okay I've fixed all the line endings

**Comment by sdogruyol** [comment]
@Shadowys thank you 👍

**Pull request merged**